sympy/stats/crv_types.py

```diff
@@ -2279,6 +2281,85 @@ def Levy(name, mu, c):
     return rv(name, LevyDistribution, (mu, c))
 
 #-------------------------------------------------------------------------------
+# Log-Cauchy distribution --------------------------------------------------------
+
+
+class LogCauchyDistribution(SingleContinuousDistribution):
+    _argnames = ('x0', 'gamma')
+
+    set = Interval(0, oo)
+
+    @property
+    def set(self):
+        return Interval(self.x0, self.b)
+
+    @staticmethod
+    def check(x0, gamma):
+        _value_check(gamma > 0, "Scale parameter Gamma must be positive.")
+        _value_check(x0.is_real, "Location parameter must be real.")
+
+    def pdf(self, x):
+        x0, gamma = self.x0, self.gamma
+        return 1/(x*pi)*(gamma/((log(x) - x0)**2 + gamma**2))
+
+    def _cdf(self, x):
+        x0, gamma = self.x0, self.gamma
+        return (1/pi)*atan((log(x) - x0)/gamma) + S.Half
+
+    def _characteristic_function(self, t):
+        raise NotImplementedError("The characteristic function for the "
+                                  "Log-Cauchy distribution does not exist.")
+
+    def _moment_generating_function(self, t):
+        raise NotImplementedError("The moment generating function for the "
+                                  "Log-Cauchy distribution does not exist.")
+
+def LogCauchy(name, x0, gamma):
+    r"""
+   Create a continuous random variable with a Log-Cauchy distribution.
+    The density of the Log-Cauchy distribution is given by
+
+    .. math::
+        f(x) := \frac{1}{\pi x} \frac{\gamma}{(log(x)-\x0^2) + \gamma^2}
+
+    Parameters
+    ==========
+
+    x0 : Real number, the location
+
+    gamma : Real number, `\gamma > 0`, a scale
+
+    Returns
+    =======
+
+    RandomSymbol
+
+    Examples
+    ========
+
+    >>> from sympy.stats import  LogCauchy, density, cdf
+    >>> from sympy import Symbol, S
+
+    >>> x0 = 2
+    >>> gamma = S.One / 5
+    >>> z = Symbol("z")
+
+    >>> X = LogCauchy("x", x0, gamma)
+
+    >>> density(X)(z)
+    1/(5*pi*z*((log(z) - 2)**2 + 1/25))
+
+    >>> cdf(X)(z)
+    atan(5*log(z) - 10)/pi + 1/2
+
+    References
+    ==========
+
+    .. [1] https://en.wikipedia.org/wiki/Log-Cauchy_distribution
```

Comment:
Please use a reference where `gamma`, `x0` notation is used. In the above link, they use, `mu` and `sigma`.

Reply:
used mu and sigma notation